ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -318,31 +318,32 @@ Control {
                                 spacing: 9
                                 Repeater {
                                     model: root.messageDetails.albumCount
-                                    Loader {
+
+                                    delegate: Loader {
```

Comment:
I see what you did there ;) Was about to report the bug :D

Reply:
Yeah I waste a lot of time to understand why its doesn't work or showing just one image for all images in album :D

Reply:
Right, I was only seeing the placeholders, never the actual images